src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1892,5 +1894,29 @@ private void checkStudyContainsNode(UUID studyUuid, UUID nodeUuid) {
             throw new StudyException(NOT_ALLOWED);
         }
     }
+
+    public Mono<Void> lineSplitWithVoltageLevel(UUID studyUuid, String lineSplitWithVoltageLevelAttributes,
+        ModificationType modificationType, UUID nodeUuid, UUID modificationUuid) {
+        return Mono.zip(getModificationGroupUuid(nodeUuid), getVariantId(nodeUuid)).flatMap(tuple -> {
+            UUID groupUuid = tuple.getT1();
+            String variantId = tuple.getT2();
+
+            Mono<Void> monoUpdateStatusResult = updateStatuses(studyUuid, nodeUuid);
```

Comment:
Pb here : you must pass an additional boolean parameter 'false' to updateStatuses method when you update an existing modification, and not when you create a new modification

Reply:
Done